README.md

```
@@ -9,20 +9,100 @@
+mindjunkies/
+├── accounts/       # User authentication and profile management
+├── classrooms/     # Classroom creation, management, and participation
+├── assignments/    # Assignment submission and evaluation
+├── video/          # Live video conferencing feature
+├── static/         # Static files (CSS, JS, images)
+├── templates/      # HTML templates
+├── config/         # Django project settings and configurations
+└── manage.py       # Django project entry point
+```
 
 ## Resources
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)